commit 50dc0486ae063eabae1f0d5a9a565a854106ab00 (refs/changes/06/2706/3)
Author: Sam Gwydir <sam@samgwydir.com>
Date:   2017-10-06T15:40:42-07:00 (2 years ago)
    
    MANATEE-371 Want ISM for SysV Shared Memory in Postgres >= 9.3
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/.gitmodules b/.gitmodules
index fc61b5d..b302921 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -6,7 +6,7 @@
 	url = https://github.com/postgres/postgres.git
 [submodule "deps/postgresql96"]
 	path = deps/postgresql96
-	url = https://github.com/postgres/postgres.git
+	url = https://github.com/joyent/postgres.git
 [submodule "deps/pg_repack"]
 	path = deps/pg_repack
 	url = https://github.com/reorg/pg_repack.git
diff --git a/deps/postgresql96 b/deps/postgresql96
index ca9cfed..563881d 160000
--- a/deps/postgresql96
+++ b/deps/postgresql96
@@ -1 +1 @@
-Subproject commit ca9cfed883333d5801716eb01cf28b6b5be2b5cd
+Subproject commit 563881d704954a95174be84330af3e218e30143b
diff --git a/etc/9.6/postgresql.manta.coal.conf b/etc/9.6/postgresql.manta.coal.conf
index 5170bb8..6d02d7b 100644
--- a/etc/9.6/postgresql.manta.coal.conf
+++ b/etc/9.6/postgresql.manta.coal.conf
@@ -125,7 +125,12 @@ shared_buffers = 32MB                   # min 128kB
 #replacement_sort_tuples = 150000       # limits use of replacement selection sort
 #autovacuum_work_mem = -1               # min 1MB, or -1 to use maintenance_work_mem
 #max_stack_depth = 2MB                  # min 100kB
-#dynamic_shared_memory_type = posix     # the default is the first option
+shared_memory_type = sysv               # the default is the first option
+                                        # supported by the operating system:
+                                        #   mmap
+                                        #   sysv
+                                        #   windows
+dynamic_shared_memory_type = sysv       # the default is the first option
                                         # supported by the operating system:
                                         #   posix
                                         #   sysv
diff --git a/etc/9.6/postgresql.manta.lab.conf b/etc/9.6/postgresql.manta.lab.conf
index 9bd60ee..6341d4a 100644
--- a/etc/9.6/postgresql.manta.lab.conf
+++ b/etc/9.6/postgresql.manta.lab.conf
@@ -125,7 +125,12 @@ maintenance_work_mem = 1GB              # min 1MB
 #replacement_sort_tuples = 150000       # limits use of replacement selection sort
 #autovacuum_work_mem = -1               # min 1MB, or -1 to use maintenance_work_mem
 #max_stack_depth = 2MB                  # min 100kB
-#dynamic_shared_memory_type = posix     # the default is the first option
+shared_memory_type = sysv               # the default is the first option
+                                        # supported by the operating system:
+                                        #   mmap
+                                        #   sysv
+                                        #   windows
+dynamic_shared_memory_type = sysv       # the default is the first option
                                         # supported by the operating system:
                                         #   posix
                                         #   sysv
diff --git a/etc/9.6/postgresql.manta.production.conf b/etc/9.6/postgresql.manta.production.conf
index 10f800d..9a650e6 100644
--- a/etc/9.6/postgresql.manta.production.conf
+++ b/etc/9.6/postgresql.manta.production.conf
@@ -125,7 +125,12 @@ maintenance_work_mem = 4GB              # min 1MB
 #replacement_sort_tuples = 150000       # limits use of replacement selection sort
 #autovacuum_work_mem = -1               # min 1MB, or -1 to use maintenance_work_mem
 #max_stack_depth = 2MB                  # min 100kB
-#dynamic_shared_memory_type = posix     # the default is the first option
+shared_memory_type = sysv               # the default is the first option
+                                        # supported by the operating system:
+                                        #   mmap
+                                        #   sysv
+                                        #   windows
+dynamic_shared_memory_type = sysv       # the default is the first option
                                         # supported by the operating system:
                                         #   posix
                                         #   sysv
